src/__stories__/JsonSchemaViewer.tsx

```diff
@@ -37,50 +39,24 @@ storiesOf('JsonSchemaViewer', module)
     <JsonSchemaViewer
       schema={object('schema', {})}
       defaultExpandedDepth={number('defaultExpandedDepth', 0)}
-      onGoToRef={action('onGoToRef')}
       maxRows={number('maxRows', 5)}
       mergeAllOf={boolean('mergeAllOf', true)}
     />
   ))
-  .add('custom row renderer', () => {
```

Comment:
Ditto.
What about masking? Masking made use of it.

Reply:
Got it, I'll add it back tomorrow. See TODO item 1.